live-examples/css-examples/backgrounds-and-borders/border-radius.css

```diff
@@ -0,0 +1,7 @@
+#example-element {
+	width: 200px;
+	height: 200px;
+	line-height: 200px;
+	background-color: #333333;
```

Comment:
I might choose a slightly brighter background color. We use #5b6dcd for `margin` and that looks quite nice here.